infrastructure-playbooks/lv-create.yml

```diff
@@ -0,0 +1,80 @@
+- name: creates logical volumes for the bucket index or fs journals on a single device and prepares for use of osd_scenario=lvm. 
+  hosts:
+  - osds
+
+  tasks:
+
+  - name: Include vars of lv_vars.yaml
+    include_vars:
+      file: lv_vars.yaml
+
+  # need to check if lvm2 is installed
+  - name: install lvm2
+    package:
+      name: lvm2
+      state: present
+
+  # Make entire nvme device a VG
+  - name: add nvme device as LVM PV
+    lvg:
+      force: yes
+      pvs: "{{nvme_device}}"
```

Comment:
Could we standardize variable usage with a space after the brackets? e.g. ``{{ nvme_device }}``

Reply:
addressed. let me know if this looks fine.